media-video/obs-studio/obs-studio-9999.ebuild

```diff
@@ -69,6 +79,21 @@ src_configure() {
 		-DOBS_MULTIARCH_SUFFIX=${libdir#lib}
 		-DUNIX_STRUCTURE=1
 	)
+
+	if use luajit || use python; then
+		mycmakeargs+=( -DENABLE_SCRIPTING=yes )
+
+		if ! use luajit; then
+			mycmakeargs+=( -DENABLE_LUA=no )
+		fi
+
+		if ! use python; then
+			mycmakeargs+=( -DENABLE_PYTHON=no )
```

Comment:
why not `-DENABLE_PYTHON=$(usex python)`? you could drop both conditionals that way.

Reply:
I had that in mind at some point, but didn't get to trying it out before I had been at it for probably way too long for one go.

I'll look into getting that change done.